autogen/oai/anthropic.py

```diff
@@ -314,7 +314,7 @@ def oai_messages_to_anthropic_messages(params: Dict[str, Any]) -> list[dict[str,
     last_tool_result_index = -1
     for message in params["messages"]:
         if message["role"] == "system":
-            params["system"] = message["content"]
+            params["system"] = params.get("system", "") + (" " if "system" in params else "") + message["content"]
```

Comment:
Shall we use "\n" as the separator in case params["system"] doesn't contain a punctuation in the end?
Do we need to remove this message from the list?

Reply:
Yep, good idea regarding the new line, will correct.

No need to remove the message from the list as messages parameter only used in this loop and no further evaluation of the role=system messages in the loop.

Reply:
Just a note I updated with new line separator.